### PR TITLE
[WFLY-12038] Expose the enable1Prefixes configuration on external (pooled) connection factories.

### PR DESCRIPTION
**Pull request by ehsavoie**
Jira: https://issues.jboss.org/browse/WFLY-12038

### CODE REVIEW

File: messaging/WFLY-12038_enable1prefixes.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+* https://issues.jboss.org/browse/ENTESB-10484[ENTESB-10484]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+TBD
```
Comment (honza-kasik) [inline messaging/WFLY-12038_enable1prefixes.adoc]: Me - jkasik

File: messaging/WFLY-12038_enable1prefixes.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFCORE[WFLY-12038]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/ENTESB-10484[ENTESB-10484]
```
Comment (honza-kasik) [inline messaging/WFLY-12038_enable1prefixes.adoc]: Can you please mention how ENTESB-10484 is related to this feature?

File: messaging/WFLY-12038_enable1prefixes.adoc [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+N/A
+
+=== Non-Requirements
+
+N/A
+
+== Test Plan
+
+Create a core queue on a remote broker without the prefixes and connect to it disabling the enable1Prefixes on the pooled-connection-factory to access it by its name.
```
Comment (honza-kasik) [inline messaging/WFLY-12038_enable1prefixes.adoc]: Some negative test scenarios would be great too.

**Comment by jamezp** [comment]
Upstream PR has been merged https://github.com/wildfly/wildfly/pull/12288.

**Pull request merged**